test/e2e/case1_spec_sync_test.go

```diff
@@ -18,14 +18,15 @@ const case1PolicyYaml string = "../resources/case1_spec_sync/case1-test-policy.y
 var _ = Describe("Test spec sync", func() {
 	BeforeEach(func() {
 		By("Creating a policy on hub cluster in ns:" + testNamespace)
-		utils.Kubectl("apply", "-f", case1PolicyYaml, "-n", testNamespace,
+		_, err := utils.KubectlWithOutput("apply", "-f", case1PolicyYaml, "-n", testNamespace,
```

Comment:
Could you please explain why this change was made in the commit message? Ideally this change would have been in a separate commit but an explanation would suffice. I'm pretty sure I know why you did it but someone without context would not and so looking at the commit message would help them.